projects/angular/src/data/tree-view/tree.ts

```diff
@@ -72,13 +72,21 @@ export class ClrTree<T> implements AfterContentInit, OnDestroy {
   }
 
   get isMultiSelectable() {
-    return this.featuresService.selectable && this.rootNodes.length > 0;
+    return this._isMultiSelectable;
   }
 
   ngAfterContentInit() {
     this.setRootNodes();
     this.subscriptions.push(
       this.rootNodes.changes.subscribe(() => {
+        if (this.featuresService.selectable && this.rootNodes.length > 0) {
+          this._isMultiSelectable = true;
+          this.renderer.setAttribute(this.el.nativeElement, 'aria-multiselectable', 'true');
+        } else {
+          this._isMultiSelectable = false;
+          this.renderer.removeAttribute(this.el.nativeElement, 'aria-multiselectable');
+        }
```

Comment:
I would extract this to a `setMultiSelectable` method.

Reply:
corrected in ca05de4